.github/workflows/camel_version.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
       matrix:
         os: [ ubuntu-latest, macos-latest, windows-latest ]
         camel-version:
-          - "4.4.5"
+          - "4.8.5"
           - "4.0.0.redhat-00036"
           - "4.4.0.redhat-00046"
```

Comment:
we can switch also to 4.8 productized version wdyt?

Reply:
I preferred to keep it separated as there were troubles with the 4.8.0 productized https://github.com/camel-tooling/camel-dap-client-vscode/pull/849 (but seems to work with 4.8.3 productized)